src/image-paste.ts

```diff
@@ -327,10 +327,12 @@ export namespace Import {
       }
 
       if (dir === '') {
-        const thisDocument = vscode.window.activeTextEditor.document
-        const adoc = new AsciidocParser(thisDocument.uri.fsPath)
-        adoc.parseText(thisDocument.getText(), thisDocument)
-        dir = adoc.document.getAttribute('imagesdir')
+        const textDocument = vscode.window.activeTextEditor.document
+        const extensionUri = vscode.Uri.file('') // won't be used anyway... needs refactoring!
+        const {document} = await new AsciidocParser(extensionUri).parseText(textDocument.getText(), textDocument)
```

Comment:
Since we don't need to convert the document, we don't need to define a valid `extensionUri`.